src/instances/piped.json

```diff
@@ -7,6 +7,7 @@
     "https://il.ax",
     "https://piped.syncpundit.com",
     "https://piped.mha.fi",
+    "https://piped.shimul.me/",
```

Comment:
Never use trailing slashes in the ridirect urls because then you get urls like `https://piped.shimul.me//watch?v=GjMmsEwIcl0` which don't work.

Reply:
@ManeraKai, Bob is right, I am reverting the PR and removing the slash. You re-merge it again later.

Reply:
@ManeraKai, I think Bob is right, I will make a new PR to fix this.

Reply:
He already made it.

Reply:
I didn't notice that. I will close my PR. Then.